docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+
+Camus includes the following features:
+    #. Automatic topic discovery
+    #. Avro schema management
+    #. Date partitioning
+
+All work is done within a single MapReduce job divided into three stages:
+    #. Setup stage:
+        Fetch available topics and partitions from Zookeeper and latest offsets from Kafka.
+    #. Hadoop stage:
+        Allocates topic pulls among a set number of MapReduce tasks. Each task does the following:
+            #. Fetch events from Kafka server and collect count statistics.
+            #. Write data to corresponding directories based on timestamps of events.
+            #. Produce count events and write to HDFS.
+            #. Store updated offsets in HDFS.
+    #. Cleanup stage:
+        Read counts from all tasks, aggregates the values,
+        and submits the results to Kafka for consumption by other systems.
+
+Quickstart 
+----------
+
+Assuming you have confluent platform installed.
+
+.. sourcecode:: bash
+
+   # Assuming that a Hadoop cluster is already deployed and you have access to it
+   # Start a testing local Kafka cluster (1 Zookeeper node, 1 Kafka node)
+   $ cd kafka
+   $ bin/zookeeper-server-start.sh config/zookeeper.properties
+   $ bin/kafka-server-start.sh config/server.properties
+   # Create testAvro topic in Kafka
+   $ bin/kafka-topics.sh --create --zookeeper localhost:2181 \
+         --topic testAvro --partitions 1 --replication-factor 1
+   $ cd ..
+
+   # Start Schema Registry. The default schema registry settings
+   # automatically work with the default settings of ZooKeeper and Kafka.
+   $ cd schema-registry
+   $ bin/schema-registry-start config/schema-registry.properties
+   $ cd ..
+
+   # Start Kafka rest proxy.
+   $ cd kafka-rest
+   $ bin/kafka-rest-start kafka-rest.properties
+   $ cd ..
+
+   # Assuming that you have hadoop on your PATH, and make sure that
+   # schema.registry.url points to the correct address
+   $ cd camus
+   $ hadoop jar confluent-camsu-0.1.0-SNAPSHOT-shaded.jar com.linkedin.camus.etl.kafka.CamusJob \
+     -P etc/camus.properties
+
+Installation
+------------
+
+Release versions are available from the `Central
+Repository <http://search.maven.org/#search|ga|1|g%3A%22io.confluent%22%20AND%20a%3A%22camus%22>`_.
+
+Deployment
+----------
+Camus can be run from the command line. You will need to set some properties either by specifying a
+properties file on the classpath using -p (filename),or an external properties file using -P
+(path to local file system, or to hdfs),
+or from the command line using -D property=value.
+If the same property is set with multiple methods,
+the order of precedence is command-line properties, external properties file and
+classpath properties file.
+
+.. sourcecode:: bash
+
+   $ hadoop jar confluent-camsu-0.1.0-SNAPSHOT-shaded.jar com.linkedin.camus.etl.kafka.CamusJob
+      -D <property=value>
+      -P <path to external properties file>
+      -p <path to properties file from classpath>
```

Comment:
Are there just a few key properties that can be setup without using the properties file, relying on defaults for everything else? If so, it's nice to provide users with the concrete, minimal command they need in order to get things running. I know we have the quickstart command above, but putting the config settings on the command line can be really helpful to users in understanding what the key things they need to setup are.